### PR TITLE
Sassywork

### LINKED ISSUE #14: Refactor CSS to Sass
Rewrite all CSS to use Sass.   

@Dunn-Austen will work through lines 1 - 81
@hfaerber will work through lines 82 - 164

### PR DESCRIPTION
**Pull request by Dunn-Austen**
#### What's this PR do?
This PR rearranges and modifies the styling files while adding further refactored scss content:
- Renames the preexisting css files to .scss and relocates them to the scss folder
- Deletes the original copy of the css file
- Links the normalize.scss and styles.scss files using (import) within the scripts.js file
- Fixes the grid layout
- Reorganizes the HTML for the grid fix, also adds three divs
- Includes minor supportive modifications/additions within the styles.scss file
- Modifies the _variables.scss files
- Adds mixins
- Adds incomplete, bare-bones structure for media query mixins

#### Where should the reviewer start?
- Start out with an npm run start to make sure the tri-column grid layout appears, then run through the scss file changes to inspect the new code.

#### What are the relevant tickets?
- This PR should resolve issue: #14
close #14

#### Questions:
- For this PR, next steps involve collectively deciding on alterations to the dashboard style & layout. Once we've chosen how we ultimately want things to appear, we would then need to modify the page stylings and then build-in the code for responsiveness.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 <head>
   <title>Activity Tracker</title>
-<!--   <link rel="stylesheet" href="normalize.css">
-  <link rel="stylesheet" href="styles.css"> -->
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="styles.css">
```
Comment (NaomiKC92) [inline src/index.html]: Since the sass code is working, we should be able to remove these lines of code permanently.

File: src/scss/_mixins.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
+
+/*# sourceMappingURL=_mixins.css.map */
```
Comment (hfaerber) [inline src/scss/_mixins.css]: Do we need this file and the _mixins.css.map files?  I'm not clear on what these do and if their essential or not.

File: src/scss/_mixins.css.map [hunk 3]
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":[],"names":[],"mappings":"","file":"_mixins.css"}
```
Comment (hfaerber) [inline src/scss/_mixins.css.map]: Do we need this file?  See note in _mixins.css

File: src/scss/_mixins.scss [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+@mixin font-size($size) {
+  font-size: $size;
+}
+
+@mixin margin($top, $right, $bottom, $left) {
+  margin-top: $top;
+  margin-right: $right;
+  margin-bottom: $bottom;
+  margin-left: $left;
+}
```
Comment (hfaerber) [inline src/scss/_mixins.scss]: I love the switch from margin variables to a margin mixin.  Much cleaner.  Great job!

File: src/scss/styles.scss [hunk 5]
```diff
@@ -32,17 +32,16 @@ canvas {
...
 }
 
 .date {
   background-color: $header-font-color;
   border-radius: 3px;
   float: right;
-  font-size: 10px;
-  margin: $margin-5px;
-  padding: 3px 5px;
+  @include font-size(10px);
```
Comment (hfaerber) [inline src/scss/styles.scss]: Would a variable or mixin be better suited for our different font sizes?  If we use the mixin in we'd still have to manually change each one correct?  Whereas variables would let us adjust all same-size font texts in one line?

I had built the mixin for font-weight but on the line of thinking above wonder if variables would be better for this styling too?

We may want to think about our missing h3 tag in tandem with this question.

File: src/css/normalize.css [hunk 6]
```diff
@@ -376,4 +376,4 @@ template {
 
 [hidden] {
   display: none;
-}
\ No newline at end of file
+}
```
Comment (hfaerber) [inline src/css/normalize.css]: Can we delete this file too now that we have the scss version of it?

File: src/index.html (hunk 1 above)
Comment (Dunn-Austen) [reply src/index.html]: Updated! Good looks in finding this.

File: src/index.html [hunk 7]
```diff
@@ -3,8 +3,6 @@
 
 <head>
   <title>Activity Tracker</title>
-  <link rel="stylesheet" href="normalize.css">
-  <link rel="stylesheet" href="styles.css">
```
Comment (hfaerber) [inline src/index.html]: Thank you!

**Pull request merged**